packages/flame/lib/src/game/game.dart

```diff
@@ -136,26 +136,22 @@ abstract class Game extends Projector {
 
   /// Converts a global coordinate (i.e. w.r.t. the app itself) to a local
   /// coordinate (i.e. w.r.t. he game widget).
-  /// If the widget occupies the whole app ("full screen" games), this operation
-  /// is the identity.
+  /// If the widget occupies the whole app ("full screen" games), or is not
+  /// attached to Flutter, this operation is the identity.
```

Comment:
most tests don't attach our games, so this change enables it to still handle events ("in memory")